apps/src/templates/AgeDialog.jsx

```diff
@@ -50,7 +47,16 @@ const styles = {
   },
 };
 
-const sessionStorageKey = 'anon_over13';
+/*
+ * SignInOrAgeDialog uses 'anon_over13' as its session storage key.
+ * We want users seeing that dialog to have to input their age, so using
+ * a different session storage key here.
+ */
+const AGE_DIALOG_SESSION_KEY = 'ad_anon_over13';
```

Comment:
I think there is a better way:
1. reuse `anon_over13` here
2. clear `anon_over13` when resetting the session 

that way, if a student is on their home computer exploring our site they don't get repeatedly re-prompted for their age, but if they go through the "hour of code" flow they will hit the session reset and then (if using dance or applab) be re-prompted for their age.

downsides of the two-session-key approach include (1) user bad of repeatedly being prompted, and (2) increased complexity as we maintain multiple identical systems for accomplishing the same thing.